kWidget/onPagePlugins/onPagePlugins.manifest.php

```diff
@@ -206,12 +206,12 @@
 				'type' => 'string'
 			),
 			'tags' => array(
-				'doc' => 'Default: "chaptering"� Providing a tag name will grab only annotations which have that tag. 

```

Comment:
@mikeweaver - lines 209, 214 and 231 looks like a mistakenly committed charcode changes. please clean it up and recommit.